operator-metadata/dist-git-files/container.yaml

```diff
@@ -1,3 +1,6 @@
 ---
 operator_manifests:
-  manifests_dir: manifests
+  enable_digest_pinning: false
+  enable_repo_replacements: false
+  enable_registry_replacements: false
```

Comment:
why these additions?

Reply:
For automated SHA pinning OSBS added some configuration fields to their service [1]. These fields are enabled by default even though the method of SHA pinning does not work for AMQ Streams and causes the operator-metadata build to crash. So we have to explicitly disable them in order to get builds through


[1] https://osbs.readthedocs.io/en/latest/users.html#replacing-pullspecs